br/backup-storage-S3.md

```diff
@@ -7,6 +7,10 @@ summary: Learn how to use BR to back up data to and restore data from Amazon S3
 
 The Backup & Restore (BR) tool supports using Amazon S3 or other Amazon S3-compatible file storages as the external storage for backing up and restoring data.
 
+> **Note:**
+>
+> To back up data to an S3 bucket with object locking enabled, you need to use TiDB v6.3.0 or later.
```

Comment:
`object lock` seems to be proprietary. https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-lock.html

Reply:
Fixed in 1bd7552